bundle/src/assembly/resources/scenarios/Barnim/output/output_config.xml

```diff
@@ -23,7 +23,7 @@
                     <entry>Updated:Slope</entry>
                     <entry>Updated:Stopped</entry>
                     <entry>Updated:RouteId</entry>
-                    <entry>Updated:RoadPosition.EdgeId</entry>
+                    <entry>Updated:RoadPosition.Connection.Id</entry>
```

Comment:
Shouldn't this be `<...>.ConnectionId`

Reply:
Either case works. `getConnectionId()` just calls `getConnection().getId()`.